2017-04-03 15:20:32 database: wrong user or password; user: "SPRINGBOOT"
org.h2.message.DbException: Wrong user name or password [28000-190]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.engine.Engine.openSession(Engine.java:106)
	at org.h2.engine.Engine.openSession(Engine.java:175)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:153)
	at org.h2.engine.Engine.createSession(Engine.java:136)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:107)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:735)
	at org.h2.server.web.WebApp.login(WebApp.java:955)
	at org.h2.server.web.WebApp.process(WebApp.java:211)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebServlet.doGet(WebServlet.java:125)
	at org.h2.server.web.WebServlet.doPost(WebServlet.java:162)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:648)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:729)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:291)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:239)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:99)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:239)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.springframework.web.filter.HttpPutFormContentFilter.doFilterInternal(HttpPutFormContentFilter.java:87)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:239)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.springframework.web.filter.HiddenHttpMethodFilter.doFilterInternal(HiddenHttpMethodFilter.java:77)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:239)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:121)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:239)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:212)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:106)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:141)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:79)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:88)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:521)
	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1096)
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:674)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1500)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.run(NioEndpoint.java:1456)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:722)
Caused by: org.h2.jdbc.JdbcSQLException: Wrong user name or password [28000-190]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 56 more
2017-04-04 11:57:03 jdbc[11]: exception
org.h2.jdbc.JdbcSQLException: Unknown data type: "VARHAR"; SQL statement:
CREATE TABLE SHIPWRECK(
	ID INT AUTO_INCREMENT,
	NAME VARCHAR(255),
	DESCRIPTION VARCHAR(2000),
	CONDITION VARHAR(255),
	DEPTH INT,
	LATTITUDE DOUBLE,
	LONGITUDE DOUBLE,
	YEAR_DISCOVERED INT
) [50004-190]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:4009)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:3872)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5808)
	at org.h2.command.Parser.parseCreate(Parser.java:4167)
	at org.h2.command.Parser.parsePrepared(Parser.java:349)
	at org.h2.command.Parser.parse(Parser.java:304)
	at org.h2.command.Parser.parse(Parser.java:276)
	at org.h2.command.Parser.prepareCommand(Parser.java:241)
	at org.h2.engine.Session.prepareLocal(Session.java:460)
	at org.h2.engine.Session.prepareCommand(Session.java:402)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.flywaydb.core.internal.dbsupport.JdbcTemplate.executeStatement(JdbcTemplate.java:238)
	at org.flywaydb.core.internal.dbsupport.SqlScript.execute(SqlScript.java:114)
	at org.flywaydb.core.internal.resolver.sql.SqlMigrationExecutor.execute(SqlMigrationExecutor.java:71)
	at org.flywaydb.core.internal.command.DbMigrate$5.doInTransaction(DbMigrate.java:284)
	at org.flywaydb.core.internal.command.DbMigrate$5.doInTransaction(DbMigrate.java:282)
	at org.flywaydb.core.internal.util.jdbc.TransactionTemplate.execute(TransactionTemplate.java:72)
	at org.flywaydb.core.internal.command.DbMigrate.applyMigration(DbMigrate.java:282)
	at org.flywaydb.core.internal.command.DbMigrate.access$800(DbMigrate.java:46)
	at org.flywaydb.core.internal.command.DbMigrate$2.doInTransaction(DbMigrate.java:207)
	at org.flywaydb.core.internal.command.DbMigrate$2.doInTransaction(DbMigrate.java:156)
	at org.flywaydb.core.internal.util.jdbc.TransactionTemplate.execute(TransactionTemplate.java:72)
	at org.flywaydb.core.internal.command.DbMigrate.migrate(DbMigrate.java:156)
	at org.flywaydb.core.Flyway$1.execute(Flyway.java:1059)
	at org.flywaydb.core.Flyway$1.execute(Flyway.java:1006)
	at org.flywaydb.core.Flyway.execute(Flyway.java:1418)
	at org.flywaydb.core.Flyway.migrate(Flyway.java:1006)
	at org.springframework.boot.autoconfigure.flyway.FlywayMigrationInitializer.afterPropertiesSet(FlywayMigrationInitializer.java:66)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1637)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1574)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:545)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:482)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:306)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:302)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:197)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:296)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:197)
	at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1054)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:829)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:538)
	at org.springframework.boot.context.embedded.EmbeddedWebApplicationContext.refresh(EmbeddedWebApplicationContext.java:118)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:764)
	at org.springframework.boot.SpringApplication.doRun(SpringApplication.java:357)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:305)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1124)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1113)
	at com.boot.App.main(App.java:16)
2017-04-04 14:39:35 database: wrong user or password; user: "SPRINGBOOT"
org.h2.message.DbException: Wrong user name or password [28000-190]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.engine.Engine.openSession(Engine.java:106)
	at org.h2.engine.Engine.openSession(Engine.java:175)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:153)
	at org.h2.engine.Engine.createSession(Engine.java:136)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:107)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:735)
	at org.h2.server.web.WebApp.login(WebApp.java:955)
	at org.h2.server.web.WebApp.process(WebApp.java:211)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebServlet.doGet(WebServlet.java:125)
	at org.h2.server.web.WebServlet.doPost(WebServlet.java:162)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:648)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:729)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:291)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:239)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:99)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:239)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.springframework.web.filter.HttpPutFormContentFilter.doFilterInternal(HttpPutFormContentFilter.java:87)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:239)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.springframework.web.filter.HiddenHttpMethodFilter.doFilterInternal(HiddenHttpMethodFilter.java:77)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:239)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:121)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:239)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:212)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:106)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:141)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:79)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:88)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:521)
	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1096)
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:674)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1500)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.run(NioEndpoint.java:1456)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:722)
Caused by: org.h2.jdbc.JdbcSQLException: Wrong user name or password [28000-190]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 56 more
2017-04-04 14:59:52 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "SHIPWRECK" not found; SQL statement:
select shipwreck0_.id as id1_0_, shipwreck0_.condition as conditio2_0_, shipwreck0_.depth as depth3_0_, shipwreck0_.description as descript4_0_, shipwreck0_.latitude as latitude5_0_, shipwreck0_.longitude as longitud6_0_, shipwreck0_.name as name7_0_, shipwreck0_.year_discovered as year_dis8_0_ from shipwreck shipwreck0_ [42102-190]
2017-04-04 15:02:09 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "SHIPWRECK" not found; SQL statement:
insert into shipwreck (id, condition, depth, description, latitude, longitude, name, year_discovered) values (null, ?, ?, ?, ?, ?, ?, ?) [42102-190]
2017-04-04 15:02:09 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "SHIPWRECK" not found; SQL statement:
insert into shipwreck (id, condition, depth, description, latitude, longitude, name, year_discovered) values (null, ?, ?, ?, ?, ?, ?, ?) [42102-190]
2017-04-04 15:02:13 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "SHIPWRECK" not found; SQL statement:
insert into shipwreck (id, condition, depth, description, latitude, longitude, name, year_discovered) values (null, ?, ?, ?, ?, ?, ?, ?) [42102-190]
2017-04-04 15:02:13 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "SHIPWRECK" not found; SQL statement:
insert into shipwreck (id, condition, depth, description, latitude, longitude, name, year_discovered) values (null, ?, ?, ?, ?, ?, ?, ?) [42102-190]
2017-04-04 15:02:20 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "SHIPWRECK" not found; SQL statement:
select shipwreck0_.id as id1_0_, shipwreck0_.condition as conditio2_0_, shipwreck0_.depth as depth3_0_, shipwreck0_.description as descript4_0_, shipwreck0_.latitude as latitude5_0_, shipwreck0_.longitude as longitud6_0_, shipwreck0_.name as name7_0_, shipwreck0_.year_discovered as year_dis8_0_ from shipwreck shipwreck0_ [42102-190]
2017-04-04 15:02:35 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "SHIPWRECK" not found; SQL statement:
insert into shipwreck (id, condition, depth, description, latitude, longitude, name, year_discovered) values (null, ?, ?, ?, ?, ?, ?, ?) [42102-190]
2017-04-04 15:02:35 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "SHIPWRECK" not found; SQL statement:
insert into shipwreck (id, condition, depth, description, latitude, longitude, name, year_discovered) values (null, ?, ?, ?, ?, ?, ?, ?) [42102-190]
2017-04-04 15:02:36 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "SHIPWRECK" not found; SQL statement:
insert into shipwreck (id, condition, depth, description, latitude, longitude, name, year_discovered) values (null, ?, ?, ?, ?, ?, ?, ?) [42102-190]
2017-04-04 15:02:36 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "SHIPWRECK" not found; SQL statement:
insert into shipwreck (id, condition, depth, description, latitude, longitude, name, year_discovered) values (null, ?, ?, ?, ?, ?, ?, ?) [42102-190]
2017-04-04 15:02:36 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "SHIPWRECK" not found; SQL statement:
insert into shipwreck (id, condition, depth, description, latitude, longitude, name, year_discovered) values (null, ?, ?, ?, ?, ?, ?, ?) [42102-190]
2017-04-04 15:02:36 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "SHIPWRECK" not found; SQL statement:
insert into shipwreck (id, condition, depth, description, latitude, longitude, name, year_discovered) values (null, ?, ?, ?, ?, ?, ?, ?) [42102-190]
2017-04-04 15:02:37 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "SHIPWRECK" not found; SQL statement:
insert into shipwreck (id, condition, depth, description, latitude, longitude, name, year_discovered) values (null, ?, ?, ?, ?, ?, ?, ?) [42102-190]
2017-04-04 15:02:37 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "SHIPWRECK" not found; SQL statement:
insert into shipwreck (id, condition, depth, description, latitude, longitude, name, year_discovered) values (null, ?, ?, ?, ?, ?, ?, ?) [42102-190]
2017-04-04 15:02:37 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "SHIPWRECK" not found; SQL statement:
insert into shipwreck (id, condition, depth, description, latitude, longitude, name, year_discovered) values (null, ?, ?, ?, ?, ?, ?, ?) [42102-190]
2017-04-04 15:02:37 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "SHIPWRECK" not found; SQL statement:
insert into shipwreck (id, condition, depth, description, latitude, longitude, name, year_discovered) values (null, ?, ?, ?, ?, ?, ?, ?) [42102-190]
2017-04-04 15:03:29 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "SHIPWRECK" not found; SQL statement:
insert into shipwreck (id, condition, depth, description, latitude, longitude, name, year_discovered) values (null, ?, ?, ?, ?, ?, ?, ?) [42102-190]
2017-04-04 15:03:29 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "SHIPWRECK" not found; SQL statement:
insert into shipwreck (id, condition, depth, description, latitude, longitude, name, year_discovered) values (null, ?, ?, ?, ?, ?, ?, ?) [42102-190]
